sdk/tests/conformance2/transform_feedback/transform_feedback.html

```diff
@@ -204,6 +209,49 @@
 
     tf = null;
     program = null;
+
+    // Check the result of the query. It should not be available yet.
+    // This constant was chosen arbitrarily to take around 1 second on
+    // one WebGL implementation on one desktop operating system. (Busy-
+    // loops based on calling Date.now() have been found unreliable.)
+    var numEarlyTests = 50000;
```

Comment:
No big reason -- the comment explains the rationale -- wanted the other test to run more quickly, so decreased the number of iterations there.